scripts/generateMappingTable.js

```diff
@@ -25,7 +26,14 @@ request.get(`https://unicode.org/Public/idna/${unicodeVersion}/IdnaMappingTable.
     const range = cells[0].split("..");
     const start = parseInt(range[0], 16);
     const end = parseInt(range[1] || range[0], 16);
-    cells[0] = [start, end];
+    cells[0] = end === start ? start : [start, end];
+
+    cells[1] = STATUS_MAPPING[cells[1]];
```

Comment:
Here I am converting directly from the status read in IdnaMappingTable to our single digit status.